A58-client-side-weighted-round-robin-lb-policy.md

```diff
@@ -0,0 +1,251 @@
+A58: `weighted_round_robin` LB policy
+----
+* Author(s): @yousukseung
+* Approver: @markdroth
+* Status: Draft
+* Implemented in: -
+* Last updated: 2023-01-24
+* Discussion at: -
```

Comment:
Please create a discussion thread for this and add the link here, as per the gRFC process.

Reply:
Done.